### PR TITLE
Fix test failure due to v1alpha becoming v1.

### PR DESCRIPTION
**Pull request by drichardson**
The make test errors being fixed are:

```
htmltest started at 08:57:46 on /tmp/website
========================================================================
site/docs/advanced/multi-cluster-allocation/index.html
  Non-OK status: 404 --- site/docs/advanced/multi-cluster-allocation/index.html --> https://github.com/googleforgames/agones/blob/master/pkg/apis/multicluster/v1alpha1/gameserverallocationpolicy.go
  Non-OK status: 404 --- site/docs/advanced/multi-cluster-allocation/index.html --> https://github.com/googleforgames/agones/blob/master/pkg/apis/multicluster/v1alpha1/gameserverallocationpolicy.go
========================================================================
✘✘✘ failed in 661.113314ms
2 errors in 99 documents
```

## Testing Done

Run `make test`.

Before change, failed with 2 errors above. After change, passes with no errors.

### CODE REVIEW

File: site/content/en/docs/Advanced/multi-cluster-allocation.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ To enable multi-cluster allocation, set `multiClusterSetting.enabled` to `true`
 NAMESPACE=<namespace>
 FLEET_NAME=<fleet name>
 
-curl https://${EXTERNAL_IP}:443/v1alpha1/gameserverallocation \
+curl https://${EXTERNAL_IP}:443/v1/gameserverallocation \
```
Comment (pooneh-m) [inline site/content/en/docs/Advanced/multi-cluster-allocation.md]: This is still v1alpha1.

File: site/content/en/docs/Advanced/multi-cluster-allocation.md [hunk 2]
```diff
@@ -10,19 +10,19 @@ description: >
...
+[GameServerAllocationPolicy](https://github.com/googleforgames/agones/blob/master/pkg/apis/multicluster/v1/gameserverallocationpolicy.go) is the CRD defined by Agones for setting multi-cluster allocation rules. In addition to cluster priority, it describes the connection information for the target cluster, including the game server namespace, agones-allocator endpoint and client K8s secrets name for redirecting the allocation request. Here is an example of setting the priority for a cluster and it's connection rules. One such resource should be defined per cluster. For clusters with the same priority, the cluster is chosen with a probability relative to its weight.
```
Comment (pooneh-m) [inline site/content/en/docs/Advanced/multi-cluster-allocation.md]: Please replace the link with `ghlink`.

File: site/content/en/docs/Advanced/multi-cluster-allocation.md [hunk 3]
```diff
@@ -10,19 +10,19 @@ description: >
...
+There may be different types of clusters, such as on-premise, and Google Kubernetes Engine (GKE), used by a game to help with the cost saving and availability. For this purpose, Agones provides a mechanism to define priorities on the clusters. Priorities are defined on [GameServerAllocationPolicy](https://github.com/googleforgames/agones/blob/master/pkg/apis/multicluster/v1/gameserverallocationpolicy.go) agones CRD. A matchmaker can enable the multi-cluster rules on a request and target [agones-allocator]({{< relref "allocator-service.md">}}) endpoint in any of the clusters and get resources allocated on the cluster with the highest priority. If the cluster with the highest priority is overloaded, the allocation request is redirected to the cluster with the next highest priority.
```
Comment (pooneh-m) [inline site/content/en/docs/Advanced/multi-cluster-allocation.md]: Please replace the link with `ghlink`.

Comment (drichardson) [reply site/content/en/docs/Advanced/multi-cluster-allocation.md]: What is ghlink?

Comment (pooneh-m) [reply site/content/en/docs/Advanced/multi-cluster-allocation.md]: You can find examples of ghlink [here](https://github.com/googleforgames/agones/search?q=ghlink&unscoped_q=ghlink)

Comment (drichardson) [reply site/content/en/docs/Advanced/multi-cluster-allocation.md]: Oh, is there some effort to convert links to ghlink? I was just following the local example doing what I thought was the minimum to make the tests pass. I'm going to be done with this PR now, so if it isn't sufficient, please close it.

Comment (pooneh-m) [reply site/content/en/docs/Advanced/multi-cluster-allocation.md]: Thanks. It is good enough. I'll add the ghlink. Thanks for fixing the make test.

File: site/content/en/docs/Advanced/multi-cluster-allocation.md [hunk 4]
```diff
@@ -96,7 +96,7 @@ To enable multi-cluster allocation, set `multiClusterSetting.enabled` to `true`
 NAMESPACE=<namespace>
 FLEET_NAME=<fleet name>
 
-curl https://${EXTERNAL_IP}:443/v1alpha1/gameserverallocation \
+curl https://${EXTERNAL_IP}:443/v1alpha/gameserverallocation \
```
Comment (drichardson) [inline site/content/en/docs/Advanced/multi-cluster-allocation.md]: @pooneh-m Uh oh, I screwed up here. I meant to revert back to v1alpha1 but used v1alpha instead.

File: site/content/en/docs/Advanced/multi-cluster-allocation.md [hunk 5]
```diff
@@ -10,19 +10,19 @@ description: >
...
+[GameServerAllocationPolicy](https://github.com/googleforgames/agones/blob/master/pkg/apis/multicluster/v1/gameserverallocationpolicy.go) is the CRD defined by Agones for setting multi-cluster allocation rules. In addition to cluster priority, it describes the connection information for the target cluster, including the game server namespace, agones-allocator endpoint and client K8s secrets name for redirecting the allocation request. Here is an example of setting the priority for a cluster and it's connection rules. One such resource should be defined per cluster. For clusters with the same priority, the cluster is chosen with a probability relative to its weight.
 
 In the following example the policy is defined for cluster B in cluster A.
 
 ```bash
 cat <<EOF | kubectl apply -f -
-apiVersion: multicluster.agones.dev/v1alpha1
+apiVersion: multicluster.agones.dev/v1
```
Comment (drichardson) [inline site/content/en/docs/Advanced/multi-cluster-allocation.md]: Also screwed this up. Meant to revert.

File: site/content/en/docs/Advanced/multi-cluster-allocation.md (hunk 4 above)
Comment (pooneh-m) [reply site/content/en/docs/Advanced/multi-cluster-allocation.md]: Thanks for the catch. I'll fix it with my change.

File: site/content/en/docs/Advanced/multi-cluster-allocation.md (hunk 5 above)
Comment (pooneh-m) [reply site/content/en/docs/Advanced/multi-cluster-allocation.md]: This is correct

**Pull request merged**